cli/fs.rs

```diff
@@ -188,3 +188,26 @@ mod tests {
     assert_eq!(resolve_from_cwd(expected).unwrap(), expected);
   }
 }
+
+pub fn files_in_subtree<F>(root: PathBuf, filter: F) -> Vec<PathBuf>
+where
+  F: Fn(&Path) -> bool,
+{
+  // TODO(ry) Use WalkDir instead of globs.
```

Comment:
Maybe even [`fs::read_dir`](https://doc.rust-lang.org/std/fs/fn.read_dir.html)?

Reply:
It's not recursive, but yes some recursive version of that.